content/SCALE/SCALETutorials/Apps/CommunityApps/TFTP-HPAApp.md

```diff
@@ -1,85 +1,114 @@
+## First Steps
 
-* Disable the TFTP service.
-  Go to **System Settings > Services** and disable the service and clear the **Start Automatically** checkbox. 
-  This prevents the service from re-enabling after a system restart.
-
-* Review your TFTP service settings and note all directory, host, auxiliary parameter, permission, and credential (username and password) settings.
+Configure DHCP server for network boot to work.
 
 To grant access to a specific user (and group) different from defaults, add the new non-root administrative user and note the UID and GID for this user.
 
 To use a specific dataset or volume for files, create this in the **Storage** screen first.
 
-## Migrating from TrueNAS TFTP Service
+## Installing the TFTP Service App
+The TFTP Service application does not require advanced preparation before installing the application.
+
+To install the **TFTP Server** app, go to **Apps**, click **Discover Apps**. Either begin typing TFTP into the search field or scroll down to locate the **TFTP Sever** application widget.
+
+{{< trueimage src="/images/SCALE/23.10/TFTPServerAppWidget.png" alt="TFTP Server Application Widget" id="1: TFTP Server Application Widget" >}}
+
+Click on the widget to open the ***TFTP Sever** information screen.
+
+{{< trueimage src="/images/SCALE/23.10/TFTPServerAppInfoScreen.png" alt="TFTP Server Information Screen" id="2: TFTP Server Information Screen" >}}
 
-After disabling the TFTP service, install the **tftpd-hpa** application.
-Go to **Apps** click on **Available Applications** and locate the **tftpd-hpa** application widget.
+Click **Install** to open the **TFTP Sever** configuration wizard.
 
-{{< trueimage src="/images/SCALE/22.12/tftpd-hpaAppWidget.png" alt="tftpd-hpa Application Widget" id="1: tftpd-hpa Application Widget" >}}
+{{< trueimage src="/images/SCALE/23.10/InstallTFTPServerAppScreen.png" alt="Install TFTP Server Screen" id="3: Install TFTP Server Screen" >}}
 
-Click **Install** to open the **tftpd-hpa** configuration wizard.
+Application configuration settings are presented in several sections. 
+To find specific fields click in the **Search Input Fields** search field, scroll down to a particular section or click on the section heading on the navigation area in the upper-right corner.
 
-{{< trueimage src="/images/SCALE/22.12/InstallTFTPdAppNameTimezone.png" alt="Install tftpd-hpa Application Name and Timezone" id="2: Install tftpd-hpa Application Name and Timezone" >}}
+You can install the application using all default settings, or you can customize setting to suit your use case. 
 
-Accept the default value or enter a name in **Application Name**.
+Click **Install** to start the installation process.
+The TFTP Server application displays on the **Installed** applications screen when the installation completes.
 
+{{< trueimage src="/images/SCALE/23.10/TFTPServerAppInstalled.png" alt="TFTP Server App Installed" id="4: TFTP Server App Installed" >}}
```

Comment:
In terms of organization, I think it would be better to move this to the bottom of the article? If I'm working through this tutorial as a user, I'm expecting it will have taken me through the relevant steps before telling me to click install.

Reply:
Rewritten to indicate they click Install after they accept or change default settings. This is in the next commit.